public/app/features/datasources/components/EmptyStateNoDatasource.tsx

```diff
@@ -68,8 +68,14 @@ export function EmptyStateNoDatasource({ onCTAClick, loading = false, title, CTA
               <ul className={styles.datasources}>
                 {datasources.map((d) => (
                   <li key={d.id}>
-                    <button onClick={() => onAddDatasource(d)}>
+                    <button
+                      onClick={() => {
+                        reportInteraction('dashboards_connectds_ds_clicked');
+                        onAddDatasource(d);
+                      }}
+                    >
                       <img
+                        role="presentation"
```

Comment:
given this image is just a presentation item (it adds no value to screen reader users, or worse, deteriorates the experience given it makes screen readers read the whole image alt text, which is not needed as the button also reads the datasource name) i marked added this. this also makes tests selector simpler.

Reply:
You can (and should!) remove the alt tag and just set it to empty string. https://www.w3.org/WAI/tutorials/images/decorative/#example-2-decorative-image-as-part-of-a-text-link

Reply:
good point! didn't think about that solution

Reply:
done in 6fa88cf. is there anything that needs to be done with the translation key?

Reply:
just run `yarn i18n:extract` and it should remove it from the english JSON, and then later Crowdin will clean up the translations.

Reply:
done. i also see a bunch of unrelated changes FYI

Reply:
yes thank you i am inspecting 😭

Reply:
may be a bit of a pain, but maybe that command should be added to the pre-commit hooks and in CI we should check if that doesn't generate any change? (or at least the latter)

Reply:
lmk if you prefer me to revert that commit and look at it separately

Reply:
Yeah, we need to add some kind of automation or check for it. It's on the list.

Feel free to keep this change in at the moment, it needs to go in nontheless. Otherwise, ill update it tomorrow.